### PR TITLE
Test that we can reach the edit-home view and back

### PR DESCRIPTION
**Pull request by drewvolz**
Adds a general test to navigate from `Home > Edit Home > Home`.

### CODE REVIEW

File: source/views/components/nav-buttons/edit-home.js [hunk 1]
```diff
@@ -18,10 +18,15 @@ export function EditHomeButton({
 }) {
 	return (
 		<Touchable
+			accessibilityComponentType="button"
+			accessibilityLabel="Open Edit Home"
```
Comment (hawkrives) [inline source/views/components/nav-buttons/edit-home.js:22]: This label doesn't feel right? I'd almost rather it was just kept as "Edit".

File: e2e/basic-smoke.spec.js [hunk 2]
```diff
@@ -18,4 +18,12 @@ describe('Basic smoke tests', () => {
 		await element(by.id('header-back')).tap()
 		await expect(element(by.id('screen-homescreen'))).toBeVisible()
 	})
+
+	it('should show home screen after tap to exit edit-home screen', async () => {
```
Comment (hawkrives) [inline e2e/basic-smoke.spec.js]: Please put this in a new file.

File: e2e/basic-smoke.spec.js [hunk 3]
```diff
@@ -18,4 +18,12 @@ describe('Basic smoke tests', () => {
...
+		await device.reloadReactNative()
+		await element(by.id('button-open-edit-home')).tap()
+		await expect(element(by.id('screen-homescreen'))).toBeNotVisible()
```
Comment (hawkrives) [inline e2e/basic-smoke.spec.js]: You don't need this one in every test; it was just me making sure that the homescreen wasn't actually visible when another screen was active.

Which should really be its own test, and not repeated in every one.

**Review by hawkrives — CHANGES_REQUESTED** [review]
nice

File: e2e/basic-smoke.spec.js (hunk 2 above)
Comment (drewvolz) [reply e2e/basic-smoke.spec.js]: This is part of the general homescreen smokescreen tests. Why?

Comment (drewvolz) [reply e2e/basic-smoke.spec.js]: We're testing out the two top buttons here.

Comment (hawkrives) [reply e2e/basic-smoke.spec.js]: It's part of the _homescreen_ smoke tests, not the _app_ smoke tests. This file is just to make sure that we can open the app and do a single navigation. Everything else should go into new files.

File: source/views/components/nav-buttons/edit-home.js (hunk 1 above)
Comment (hawkrives) [reply source/views/components/nav-buttons/edit-home.js:22]: Okay, this makes more sense. Nevermind. (I'm now playing with voiceover.)

File: e2e/basic-smoke.spec.js (hunk 2 above)
Comment (drewvolz) [reply e2e/basic-smoke.spec.js]: What would you like the new file named, and what else would happen in the new set of tests?

File: e2e/basic-smoke.spec.js (hunk 3 above)
Comment (drewvolz) [reply e2e/basic-smoke.spec.js]: That's fair.

File: e2e/basic-smoke.spec.js (hunk 2 above)
Comment (hawkrives) [reply e2e/basic-smoke.spec.js]: "e2e/views/homescreen/edit.spec.js"?

And for now, just this one; in the future, things like testing that we can drag and drop the rows (not yet possible in Detox) or ensuring that flipping a toggle switch hides the view from the homescreen. Or, in e2e/views/homescreen/home.js, making sure that you can scroll to the bottom on smaller devices.

**Review by hawkrives — DISMISSED** [review]

**Review by hawkrives — APPROVED** [review]

**Pull request merged**